test-lib-openshift.sh

```diff
@@ -273,6 +273,13 @@ function ct_os_new_project() {
   oc new-project ${project_name}
   # let openshift cluster to sync to avoid some race condition errors
   sleep 3
+  if test -n "${OPENSHIFT_CLUSTER_PULLSECRET_PATH:-}" -a -e "${OPENSHIFT_CLUSTER_PULLSECRET_PATH:-}"; then
+    oc create -f "$OPENSHIFT_CLUSTER_PULLSECRET_PATH"
+    # add registry pullsecret to the serviceaccount if provided
+    secret_name=$(grep "name:" "$OPENSHIFT_CLUSTER_PULLSECRET_PATH" | awk '{ print $2 }')
```

Comment:
I'd suggest to use '^\s*name:' to avoid a potential wrong match in the future.

Reply:
Makes sense, will fix